scripts/west_commands/ncs_commands.py

```diff
@@ -505,11 +505,14 @@ def _name_and_path(project):
      'modules/hal/openisa',
      'modules/hal/qmsi',
      'modules/hal/quicklogic',
+     'modules/hal/renesas',
      'modules/hal/rpi_pico',
      'modules/hal/silabs',
      'modules/hal/stm32',
      'modules/hal/telink',
      'modules/hal/ti',
+     'modules/hal/wurthelektronik',
```

Comment:
This must be enabled (it's a sensor HAL), can be done in a follow-up PR. Remember to include it in the `allowlist` of the import.

Reply:
Addressed in https://github.com/nrfconnect/sdk-nrf/pull/9082.